index.md

```diff
@@ -1,27 +1,84 @@
-# Helm Repository for Mayastor
+# OpenEBS Helm Repository
 
-## Installation Guide
+<img width="200" align="right" alt="OpenEBS Logo" src="https://raw.githubusercontent.com/cncf/artwork/master/projects/openebs/stacked/color/openebs-stacked-color.png" xmlns="http://www.w3.org/1999/html">
+
+[OpenEBS](https://openebs.io) helps Developers and Platform SREs easily deploy Kubernetes Stateful Workloads that require fast and highly reliable container attached storage. OpenEBS can be deployed on any Kubernetes cluster - either in cloud, on-premise (virtual or bare metal) or developer laptop (minikube).
+
+OpenEBS Data Engines and Control Plane are implemented as micro-services, deployed as containers and orchestrated by Kubernetes itself. An added advantage of being a completely Kubernetes native solution is that administrators and developers can interact and manage OpenEBS using all the wonderful tooling that is available for Kubernetes like kubectl, Helm, Prometheus, Grafana, etc.
+
+OpenEBS turns any storage available on the Kubernetes worker nodes into local or distributed Kubernetes Persistent Volumes.
+* Local Volumes are accessible only from a single node in the cluster. Pods using Local Volume have to be scheduled on the node where volume is provisioned. Local Volumes are typically preferred for distributed workloads like Cassandra, MongoDB, Elastic, etc that are distributed in nature and have high availability built into them. Depending on the type of storage attached to your Kubernetes worker openebs offers different flavors of Local PV - Hostpath, LVM and  ZFS.
+* Replicated Volumes as the name suggests, are those that have their data synchronously replicated to multiple nodes. Volumes can sustain node failures. The replication also can be setup across availability zones helping applications move across availability zones. OpenEBS offers MayaStor as an replicated storage solution, which provides high availability and high performance.
+
+## Documentation and user guides
+
+You can run OpenEBS on any Kubernetes 1.23+ cluster in a matter of minutes. See the [Quickstart Guide to OpenEBS](https://openebs.io/docs/quickstart-guide/installation) for detailed instructions.
+
+## Getting started
+
+### How to customize OpenEBS Helm chart?
+
+OpenEBS helm chart is an umbrella chart that pulls together engine specific charts. The engine charts are included as dependencies. 
+arts/openebs/Chart.yaml). 
+
+```bash
+openebs
+├── (default) LocalPV HostPath
+├── (default) LocalPV LVM
+├── (default) LocalPV LVM
+└── (default) MayaStor (replicated)
```

Comment:
Make this consistent with the docs? CC @avishnu 
EDIT: oh there's LVM twice :)

Reply:
Sorry did not understand, consistent in terms of which docs?

Reply:
I have fixed that.

Reply:
That's why I asked about the docs, should be consistent?

Reply:
Yeah. Should be consistent. @avishnu are we going with `ReplicatedPV Mayastor` ?

Reply:
For now, Mayastor in this PR

Reply:
Done.

Reply:
As discussed, let's change to the following:
├── (default) Local PV HostPath
├── (default) Local PV LVM
├── (default) Local PV LVM
└── (default) Replicated PV Mayastor

Reply:
Fixed.